### PR TITLE
Npm audit fix

### PR DESCRIPTION
**Pull request by Milosdanjov**
**Before**
Found 18 vulnerabilities (4 low, 2 moderate, 12 high)

**After** 
3 low severity vulnerabilities.

Pls. check the image bellow, it says like  `No fix available`, maybe have someone an idea what should I do? (FYI I tried to install yargs-parser to `v- 15.0.0` but it didn't help me..)

I got this after npm audit fix --force

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
   "homepage": "https://github.com/autoricardo/carforyou-phraseapp-pkg#readme",
   "devDependencies": {
     "@carforyou/eslint-config": "3.1.7",
-    "@pika/pack": "^0.5.0",
+    "@pika/pack": "^0.3.7",
```
Comment (masone) [inline package.json:19]: what's the motivation for the downgrade here? the build scripts fail because of it

Comment (Milosdanjov) [reply package.json:19]: that's changed after `npm audit fix --forces`, there was some vulnerabilities and this is the reason why it's downgraded

Comment (masone) [reply package.json:19]: I see. that's what the --force does. I would not downgrade pika though. let's stick to what the non-forced fix can do and wait for a fix in pika. these are build-time dependencies, so they would not be a risk for a production deployment

**Comment by masone** [comment]
as discussed on a call, I would start over. we're having the same issue locally on the branch, but it works on master. somehow, the packagelock seems to be inconsistent

**Comment by Milosdanjov** [comment]
I'll close this PR and create a new one, because of some error on my local branch. 
(Already discussed with Christian about it)

**Comment by Milosdanjov** [comment]
I'll close this PR and create a new one, because of some error on my local branch. 
(Already discussed with Christian about it)

**Pull request closed without merging**